backend/app/models/reading_list.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: reading_lists
+#
+#  id         :uuid             not null, primary key
+#  status     :string           default("unread")
```

Comment:
`nit` since this is a practice app. But this is a column with a finite number of values. Values that your app logic will depend on (e.g. if reading_list.status == :active). 

For these kinds of columns, I'd recommend an [enum](https://blog.saeloun.com/2022/01/05/how-to-use-enums-in-rails.html) to ensure that your model, and your database, are both strict to the values it can have.